test/testscripts/hvband.jl

```diff
@@ -0,0 +1,21 @@
+using Colors, Gadfly, RDatasets
+set_default_plot_size(21cm, 8cm)
+
+Dp = dataset("ggplot2","presidential")[3:end,:]
+De = dataset("ggplot2","economics")
+De[:Unemploy] /= 10^3
```

Comment:
it's great to have nice examples like this that use real data sets in the documentation, but for tests i think it is better to have very simple test cases like in `testscripts/rect.jl`.  easier to debug if necessary, will execute more quickly, and create smaller files.